src/FlubuCore/Infrastructure/FlubuConsoleLogger.cs

```diff
@@ -236,45 +235,5 @@ private ConsoleColors GetLogLevelConsoleColors(LogLevel logLevel)
                     return new ConsoleColors(_defaultConsoleColor, _defaultConsoleColor);
             }
         }
-
-        public struct ConsoleColors
```

Comment:
I moved this class to Infrastructure/ColorfulConsole/ConsoleColors.cs.

Reply:
I think everyhing should be moved from namespace
 `Infrastructure.Terminal.ColorfulConsole` to  `Infrastructure.ColorfulConsole` or maybe even better to  `Infrastructure.Logging.ColorfulConsole` . 

`Infrastructure.Terminal` namespace is ment for Flubu interactive mode`

Reply:
> `Infrastructure.Terminal` namespace is ment for Flubu interactive mode`

Thanks for your suggestion.
I've changed namespace to `Infrastructure.ColorfulConsole` because I think those types in `ColorfulConsole` are not only for logging but mainly for console output.